tests/migrations/test_state.py

```diff
@@ -160,6 +160,23 @@ class Meta:
         self.assertEqual([mgr.args for name, mgr in food_order_manager_state.managers],
                          [('a', 'b', 1, 2), ('x', 'y', 3, 4)])
 
+    def test_apps_bulk_update(self):
+        """
+        Tests that StateApps.bulk_update() sets apps.ready to False and resets the value afterwards.
```

Comment:
Please wrap docstring at 79 chars
I prefer to use this style, "StateApps.bulk_update() should update apps.ready to False and reset the value afterwards." rather than "Tests that..." in every docstring.